dom0-updates/qubes-dom0-update

```diff
@@ -201,9 +206,17 @@ qvm-run --nogui -q $UPDATEVM 'rm -rf /var/lib/qubes/dom0-updates/etc' || exit 1
 tar c /var/lib/rpm /etc/yum.repos.d /etc/yum.conf /etc/dnf/dnf.conf 2>/dev/null | \
    qvm-run --nogui -q --pass-io "$UPDATEVM" 'LC_MESSAGES=C tar x -C /var/lib/qubes/dom0-updates 2>&1 | grep -v -E "s in the future"'
 
-qvm-run $QVMRUN_OPTS --pass-io $UPDATEVM "script --quiet --return --command '/usr/lib/qubes/qubes-download-dom0-updates.sh --doit --nogui $(escape_args "${ALL_OPTS[@]}")' /dev/null" < /dev/null
+CMD="/usr/lib/qubes/qubes-download-dom0-updates.sh --doit --nogui $(escape_args "${ALL_OPTS[@]}")"
```

Comment:
Just to be sure, have you verified the escaping? I think it is fine because `printf %q` (so `escape_args`) really escape special characters with `\`, not quote them, but worth double checking.
Most likely it will break for literal `'` inside argument (because you can't escape it with `\` within `'`), but I don't think we need to worry about it, it's very unlikely to be part of valid command line. And commands run in dom0 are trusted.

Reply:
Yeah... from my experiments: spaces are passed correctly (they reach remote `dnf`), `"` also, and `'` break in remote bash.

If we will have more situations like this, I would replace `script` with some other way of ensuring terminal over qrexec. As a bonus, we could make `qvm-run bash` more comfortable. :)